imagetest/utils/test_utils.go

```diff
@@ -105,6 +109,38 @@ func GetMetadataHTTPResponse(path string) (*http.Response, error) {
 	return resp, nil
 }
 
+// PutMetadataGuestAttribute sets the guest attribute in the namespace, and returns an error if this operation fails.
+func PutMetadataGuestAttribute(namespace, attribute string) error {
+	return PutMetadata("guest-attributes/" + namespace + "/" + attribute)
+}
+
+// PutMetadata returns a metadata value for the specified key if it is present, and error if not.
+func PutMetadata(path string) error {
+	resp, err := PutMetadataHTTPResponse(path)
+	if err != nil {
+		return err
+	}
+	if resp.StatusCode != 200 {
+		return fmt.Errorf("http response code is %v", resp.StatusCode)
+	}
+	return nil
+}
+
+// PutMetadataHTTPResponse returns http response for the specified key without checking status code.
+func PutMetadataHTTPResponse(path string) (*http.Response, error) {
```

Comment:
do we have a context we could use here?

Reply:
Huum actually we do, in the wrapper's `main()`, we have this:

```
	ctx := context.Background()
	client, err := storage.NewClient(ctx)
```

Please change this `PutMetadataHTTPResponse()` function and pass down the context.

No need to address it in this PR but we are not handling signals in the wrapper, ideally we should, the sig handler would have to cancel the context and nicely terminate the program - even if we don't expect to have the program being terminated or aborted it's a good practice.

Reply:
currently passed it down, bug created